pkgs/tools/package-management/nix-index/wrapper.nix

```diff
@@ -12,8 +11,4 @@ symlinkJoin {
     wrapProgram $out/bin/nix-index \
       --prefix PATH : ${lib.makeBinPath [ nix ]}
   '';
-
-  meta = nix-index-unwrapped.meta // {
-    hydraPlatforms = [];
-  };
```

Comment:
Don't we still want to keep `nix-index-unwrapped`'s `meta`?

Also, why build on Hydra? `symlinkJoin` sets `allowSubstitutes` to `false` by default.

Reply:
> Don't we still want to keep nix-index-unwrapped's meta?

Yeah, added that back.

> Also, why build on Hydra?

Because we only exclude it when there is a compelling reason or licensing does not allow it.

Reply:
OK, noted